vmdb/lib/workers/worker_base.rb

```diff
@@ -150,10 +151,15 @@ def log_prefix
   #
 
   def find_worker_record
-    @worker = self.class.corresponding_model.find_by_guid(@cfg[:guid])
+    @worker = self.class.corresponding_model.includes(:miq_server).find_by_guid(@cfg[:guid])
     do_exit("Unable to find instance for worker GUID [#{@cfg[:guid]}].", 1) if @worker.nil?
   end
 
+  def find_server_record
+    find_worker_record unless @worker
```

Comment:
How can `@worker` be nil?

1. [`initialize` calls `worker_initialization`](https://github.com/ManageIQ/manageiq/blob/c1b4d9d085111e2b9fdad38f81ceeb39cb3ce3cd/vmdb/lib/workers/worker_base.rb#L51)
2. [`worker_initialization` calls `starting_worker_record`](https://github.com/ManageIQ/manageiq/blob/c1b4d9d085111e2b9fdad38f81ceeb39cb3ce3cd/vmdb/lib/workers/worker_base.rb#L58)
3. [`starting_worker_record` calls `find_worker_record`](https://github.com/ManageIQ/manageiq/blob/c1b4d9d085111e2b9fdad38f81ceeb39cb3ce3cd/vmdb/lib/workers/worker_base.rb#L158)
4. [`find_worker_record` seems to blow up if there is no worker found](https://github.com/ManageIQ/manageiq/blob/c1b4d9d085111e2b9fdad38f81ceeb39cb3ce3cd/vmdb/lib/workers/worker_base.rb#L154)

It seems like you wouldn't be able to initialize this object without a worker.  Is something setting it to nil? Is some subclass not calling `super` in initialize?